app/models/test_track/misconfiguration_notifier.rb

```diff
@@ -1,13 +1,33 @@
-class TestTrack::MisconfigurationNotifier
-  def notify(msg)
-    raise msg if Rails.env.development?
+module TestTrack::MisconfigurationNotifier
+  class Wrapper
+    attr_reader :notifier
 
-    Rails.logger.error(msg)
+    def initialize(notifier = Null.new)
+      @notifier = notifier
+    end
+
+    def notify(msg)
+      raise msg if Rails.env.development?
+
+      Rails.logger.error(msg)
+
+      notifier.notify(msg)
+    end
+  end
+
+  class Null
+    def notify(_); end
+  end
 
-    if Airbrake.respond_to?(:notify_or_ignore)
-      Airbrake.notify_or_ignore(StandardError.new, error_message: msg)
-    else
-      Airbrake.notify(StandardError.new, error_message: msg)
+  class Airbrake
+    def notify(msg)
+      if defined?(::Airbrake)
```

Comment:
should this switch to a raisey guard clause? At this point you will have configured the Airbrake notifier, so if it isn't there we got problems, right?